<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>688</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>861</width>
    <height>688</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>861</width>
    <height>688</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CVCourse</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ComputeButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>250</y>
      <width>116</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute DisparityMap</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>201</width>
      <height>211</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>85</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimum Disparity</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>171</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Maximum Disparity (divisible by 16)</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>168</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Window Size (positive odd number)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="MaxDisparity">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="maximum">
      <number>320</number>
     </property>
     <property name="singleStep">
      <number>16</number>
     </property>
     <property name="value">
      <number>32</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="MinDisparity">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
     </property>
     <property name="minimum">
      <number>-320</number>
     </property>
     <property name="maximum">
      <number>320</number>
     </property>
     <property name="value">
      <number>-3</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="WindowSize">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>9</number>
     </property>
    </widget>
    <widget class="QPushButton" name="LeftLoadButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>83</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>LoadLeftImage</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RightLoadButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>86</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>LoadRightImage</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>52</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Left Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>58</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Right Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>104</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenCV DisparityMap</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>112</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Self made DisparityMap</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
